elide-annotations/src/main/java/com/yahoo/elide/annotation/Include.java

```diff
@@ -32,4 +32,10 @@
      * @return the string
      */
     String type() default "";
+
+    /**
+     * (Optional) Whether or not this class inherits permissions from a super class.
+     * @return
+     */
+    boolean inheritPermissions() default true;
```

Comment:
Do we want to invert this? If we expect micro-services to be the norm this probably wants to be `false`, because the default should be what we expect people to use in the normal case. We shouldn't default to `true` just because defaulting to `false` would be a breaking change.